projects/dynamicforms/src/core/services/data/form-data.service.ts

```diff
@@ -11,12 +12,24 @@ import { StateWithForm } from '../../store/state';
 @Injectable()
 export class FormDataService {
   submittedForm = new BehaviorSubject<YFormData>(null);
+  formGroupSource = new Subject();
+  formGroup = this.formGroupSource.asObservable();
```

Comment:
Perhaps rename this to `getFormGroupModel$` or something similar to indicate that it will return form group model and that it's an observable.